scrapers/src/extractors/office.rs

```diff
@@ -0,0 +1,321 @@
+    let matchers = MATCHERS.get_or_init(|| {
+        [
+            (
+                r"(?i:U(?:nited |.)S(?:tates|.) Senat(?:e|or))",
+                OfficeMeta {
+                    name: "U.S. Senate".into(),
+                    title: "U.S. Senator".into(),
+                    r#type: None,
+                    chamber: Some(db::Chamber::Senate),
+                    district_type: None,
+                    political_scope: db::PoliticalScope::Federal,
+                    election_scope: db::ElectionScope::State,
+                },
+            ),
+            (
+                r"(?i:U(?:nited |.)S(?:tates|.) (?:House|Representative))",
+                OfficeMeta {
+                    name: "U.S. House".into(),
+                    title: "U.S. Representative".into(),
+                    r#type: None,
+                    chamber: Some(db::Chamber::House),
+                    district_type: Some(db::DistrictType::UsCongressional),
+                    political_scope: db::PoliticalScope::Federal,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:State Senat(?:e|or))",
+                OfficeMeta {
+                    name: "State Senate".into(),
+                    title: "State Senator".into(),
+                    r#type: None,
+                    chamber: Some(db::Chamber::Senate),
+                    district_type: Some(db::DistrictType::StateHouse),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:State (?:House|Representative))",
+                OfficeMeta {
+                    name: "State House".into(),
+                    title: "State Representative".into(),
+                    r#type: None,
+                    chamber: Some(db::Chamber::House),
+                    district_type: Some(db::DistrictType::StateHouse),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:Board of Education)",
+                OfficeMeta {
+                    name: "Board of Education".into(),
+                    title: "Board of Education".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: Some(db::DistrictType::School),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:(?:Board of Regents|Regent of (?:the )?University))",
+                OfficeMeta {
+                    name: "Board of Regents".into(),
+                    title: "Board of Regents".into(), // TODO - Is this right?
```

Comment:
not sure on this one we could perhaps use
```
name: "Board of Regents"
title "Regent"
```
but tbd, can keep as is for now.

for the Board of Education above:
```
name: "Board of Education"
title: "Board of Education Member"
```

this distinction isn't particularly important, just sometimes useful for display purposes

Reply:
Sounds good, thank you!